java/src/jmri/jmrit/display/layoutEditor/LayoutTurnout.java

```diff
@@ -1115,16 +1122,16 @@ public Object getConnection(int location) throws jmri.JmriException {
             default:
                 // fall out
         }
-        log.error("Invalid Point Type " + location); //I18IN
+        log.error("Invalid Point Type " + connectionType); //I18IN
```

Comment:
Why this error and this exception are being thrown will be much clearer in the default part of the switch statement.